packages/aws-cdk-lib/core/lib/custom-resource-provider/custom-resource-provider.ts

```diff
@@ -132,5 +138,14 @@ function customResourceProviderRuntimeToString(x: CustomResourceProviderRuntime)
       return 'nodejs16.x';
     case CustomResourceProviderRuntime.NODEJS_18_X:
       return 'nodejs18.x';
+    case CustomResourceProviderRuntime.NODEJS_20_X:
```

Comment:
Can we either add deprecation to the older runtimes or remove them completely as you can no longer create or update functions that contain them (at least for 12)? https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-deprecated

Reply:
They're marked as deprecated on the `CustomResourceProviderRuntime` enum.